static/admin/config.yml

```diff
@@ -28,19 +28,39 @@ collections: # A list of collections the CMS should be able to edit
                 label: Title
                 widget: string
 
-              - name: description
-                label: Description
```

Comment:
If you are going to change this you are going to have to change it in the components themselves. You should confine the scope of this PR to the copy of the home page only and do this in a separate PR